x-pack/legacy/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -328,7 +376,7 @@ function formatEnvelopeAsPolygon({ maxLat, maxLon, minLat, minLon }) {
   const bottomRight = [right, bottom];
   const topRight = [right, top];
   return {
-    'type': 'polygon',
+    'type': GEO_JSON_TYPE.POLYGON,
```

Comment:
thanks for cleanup. So ES will handle this correctly then? (there was an issue with casing for geojson-geometry being all lower-case in ES, but i don't remember the details)

Reply:
ES supports both lower case and upper case. All other geospatial libraries only support upper case. To verify this works, just save a map. The map saved object saves the extent as a geo_shape. This is done so one day we could have a map listing page that shows all maps on a map.